vllm/v1/attention/backends/gdn_attn.py

```diff
@@ -131,6 +165,80 @@ def __init__(
             device=device,
         )
 
+        if self.vllm_config.cache_config.enable_prefix_caching:
+            self.state_indices_tensor_d_buf = torch.empty(
+                (self.decode_cudagraph_max_bs, self._max_cached_blocks),
+                dtype=torch.int32,
+                device=device,
+            )
+            self.state_indices_tensor_p_buf = torch.empty(
+                (self.decode_cudagraph_max_bs, self._max_cached_blocks),
+                dtype=torch.int32,
+                device=device,
+            )
+            self.block_idx_last_computed_token_d_buf = torch.empty(
+                (self.decode_cudagraph_max_bs,),
+                dtype=torch.int32,
+                device=device,
+            )
+            self.block_idx_last_scheduled_token_d_buf = torch.empty(
+                (self.decode_cudagraph_max_bs,),
+                dtype=torch.int32,
+                device=device,
+            )
+
+            max_num_prefill_chunks = (
+                cdiv(vllm_config.model_config.max_model_len, self.chunk_size)
+                * self.decode_cudagraph_max_bs
+            )
```

Comment:
Not sure why the number of prefill chunks could be related to the maximum decode-only batch size